GameData/RealismOverhaul/RO_SuggestedMods/RealScaleBoosters/RO_RSB_Delta.cfg

```diff
@@ -714,15 +697,45 @@
         }
     }
 
-    !MODULE[ModuleSPU]{}
+    !MODULE[ModuleSPU*],*{}
 
-    !MODULE[ModuleRTAntenna*]{}
+    !MODULE[ModuleRTAntenna*],*{}
 
     !RESOURCE,*{}
+
+    //  Avionics batteries 1.4 kWh.
+    //  Can support the DCSS for flights up to 7 hours in duration.
+    //  Assumes that the electricity consumption of the DCSS avionics is 200 W.
+
+    RESOURCE
```

Comment:
I had them originally like that but theoretiacally this requires a "ServiceModule" tank and because the parts are not modular then the DCSS tanks were also the same type (really bad!).

Other thoughts on that:

* A user might remove the tanks, rebuild them but forget the batteries (not so easy to check because of the nature of the part assembly).
* No actual reason to remove the EC since it is 100% required to operate the stage (apart from adding separate battery modules).